### PR TITLE
Make tests run with EI and MI

### PR DESCRIPTION
**Pull request by niruhan**
## Purpose
$subject

### CODE REVIEW

File: src/test/resources/automation.xml [hunk 1]
```diff
@@ -227,10 +227,21 @@
         <!--<className>org.wso2.carbon.automation.extensions.servers.carbonserver.CarbonServerExtension</className>-->
         <platformExecutionManager>
             <extentionClasses>
+                <class>
+                    <name>org.wso2.connector.integration.test.base.CopyConnectorZipToResources</name>
+                </class>
                 <class>
                     <name>org.wso2.esb.integration.common.extensions.carbonserver.CarbonServerExtension</name>
                     <parameter name="-DportOffset" value="200" />
+                    <!--For EI-->
```
Comment (abeykoon) [inline src/test/resources/automation.xml:236]: Shall we say

 
<parameter name="managementPort" value="9763" />
<parameter name="startupScript" value="integrator" />

Comment (Nirothipan) [reply src/test/resources/automation.xml:236]: can't we add a profile for EI or MI and set these as system properties when we run mvn clean install -P profile .. 
Then only the execution command will change , and we don't need to edit the files.

File: src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/fileconnector.xml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+                        <address>{$ctx:address}</address>
+                        <append>{$ctx:append}</append>
+                    </fileconnector.send>
+                </case>
+            </switch>
+            <respond/>
+        </inSequence>
+    </target>
+    <description/>
+</proxy>
```
Comment (Nirothipan) [inline src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/fileconnector.xml:262]: EOF

File: pom.xml [hunk 3]
```diff
@@ -27,12 +27,45 @@
...
+            <id>EI7XX</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>EI7XX</value>
+                </property>
+            </activation>
+            <properties>
+                <product.name>wso2mi</product.name>
+                <product.version>1.2.0-beta</product.version>
```
Comment (abeykoon) [inline pom.xml:57]: For what we need this version number? 
This will change from one pack to another.

File: pom.xml [hunk 4]
```diff
@@ -27,12 +27,45 @@
...
+    </profiles>
     <properties>
         <connector.name>fileconnector</connector.name>
-        <product.ei.version>6.6.0</product.ei.version>
         <carbon.kernel.version>4.4.38</carbon.kernel.version>
-        <automation.framework.utils.version>4.4.2</automation.framework.utils.version>
-        <automation.framework.version>4.4.3</automation.framework.version>
+        <automation.framework.version>4.5.0</automation.framework.version>
+        <integration.base.version>1.0.5-SNAPSHOT</integration.base.version>
+        <automation.extension.version>1.3.0-SNAPSHOT</automation.extension.version>
```
Comment (abeykoon) [inline pom.xml:68]: It is not wise to depend on SNAPSHOT versions for dependencies.

**Pull request merged**